cmd/glasskube/cmd/update.go

```diff
@@ -29,13 +33,22 @@ var updateCmd = &cobra.Command{
 		updater := update.NewUpdater(client).
 			WithStatusWriter(statuswriter.Spinner())
 
-		tx, err := updater.Prepare(ctx, packageNames)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "❌ update preparation failed: %v\n", err)
-			os.Exit(1)
-		}
+		var tx *update.UpdateTransaction
+		var err error
 
-		printTransaction(*tx)
+		if len(args) == 1 && updateCmdOptions.Version != "" {
+			tx, err = updater.PrepareForVersion(ctx, args[0], updateCmdOptions.Version)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error in updating the package version : %v", err)
+				os.Exit(1)
+			}
+		} else {
```

Comment:
In case the user enters more than one package and the `-v` flag, I think it would be nice to show them an error message: "Version flag not supported for multiple packages" or something like that. And after that, `os.Exit`.

Reply:
![image](https://github.com/glasskube/glasskube/assets/122217807/d4d1f755-4e56-4a61-b18e-0d4cb22e357d)
Do we need something like this?

Reply:
Fine by me, but I would remove the " : <nil>" part